src/main/resources/application.yml

```diff
@@ -118,8 +118,7 @@ redis:
 
 data-grid:
   address: localhost:7379
-  lock-time-to-wait-seconds: 30
-  lock-time-to-live-seconds: 600
+  lock-time-to-live-seconds: 3600
```

Comment:
Does this need changing in the application-cloud.yml aswell?

Reply:
Yes. Good shout